phpunit/tests/fonts/font-library/wpRestFontFamiliesController/deleteItem.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Test WP_REST_Font_Families_Controller::delete_item().
+ *
+ * @package WordPress
+ * @subpackage Fonts
+ *
+ * @group fonts
+ * @group font-library
+ *
+ * @covers WP_REST_Font_Families_Controller::delete_item
+ */
+class Tests_Fonts_WPRESTFontFamiliesController_DeleteItem extends WP_REST_Font_Families_Controller_UnitTestCase {
```

Comment:
Can we consolidate these `Tests_Fonts_WPRESTFontFamiliesController_*` test classes into one larger `WP_Test_REST_Font_Families_Controller` class?

Reply:
I suppose we could... what's the benefit? The `create_item` tests especially are pretty big and combining them into a single file seems unnecessary.  Other tests in the suite are broken similarly.  Would you want to consider consolidation of those too?